swift/ql/test/TestUtils.qll

```diff
@@ -1,6 +1,10 @@
 private import codeql.swift.elements
 private import codeql.swift.generated.ParentChild
 
+// Internal classes are not imported by the tests:
```

Comment:
I didn't manage to amend the ql_test*.mustache templates to output the full import path in case of `@ql.internal`-marked classes, but that's what should ideally replace this.